data/config-schema/build.gradle

```diff
@@ -2,6 +2,7 @@ plugins {
     id 'corda-api.common-library'
     id 'corda.common-publishing'
     id 'corda.java-only'
+    id 'corda.api-scanner'
```

Comment:
I don't think this project needs to be included.
The JSON schemas themselves are what is public API.